core/state_processor_test.go

```diff
@@ -669,6 +681,10 @@ func TestProcessVerkleInvalidContractCreation(t *testing.T) {
 					Balance: big.NewInt(1000000000000000000), // 1 ether
 					Nonce:   1,
 				},
+				params.HistoryStorageAddress: GenesisAccount{
```

Comment:
Ideally we should add it with the right code, correct?
Maybe irrelevant for these tests.

Reply:
iirc, we decided to use the code for mainnet, but in the spec it's fine as long as the account exists (which it will with nonce 1).